Makefile

```diff
@@ -223,7 +223,7 @@ release: build archive
 ################################################################################
 .PHONY: test
 test: test-deps
-	gotestsum --jsonfile $(TEST_OUTPUT_FILE_PREFIX)_unit.json --format standard-quiet -- ./pkg/... ./utils/... ./cmd/... $(COVERAGE_OPTS)
+	gotestsum --jsonfile $(TEST_OUTPUT_FILE_PREFIX)_unit.json --format standard-quiet -- -gcflags=-l ./pkg/... ./utils/... ./cmd/... $(COVERAGE_OPTS)
```

Comment:
Why add this flag?

Reply:
Hi @daixiang0  i use "bou.ke/monkey" library for mock "ioutil.ReadFile" in the unittest, if not add this flag the monkey library not working.